docs/zh-CN/administrator-guide/block-rule/sql-block.md

```diff
@@ -49,7 +49,7 @@ under the License.
 CREATE SQL_BLOCK_RULE test_rule 
 PROPERTIES(
   "sql"="select * from order_analysis",
-  "global":"false",
+  "global"="false",
```

Comment:
This pr is same to #8106

Reply:
@zyw8136 pr #8106 has been approved. 
And this pr is repeated with #8106 maybe you can close this later.

Reply:
o....
Others in the group got in first

Reply:
@spaces-X  What do I need to do? Do I need to close this pr?

Reply:
Yes, you can close this pr now.